services-directions/src/main/java/com/mapbox/api/directions/v5/DirectionsService.java

```diff
@@ -46,7 +45,13 @@
    * @param voiceUnits         voice units
    * @param exclude            exclude tolls, motorways or more along your route
    * @param approaches         which side of the road to approach a waypoint
-   * @param waypointNames     custom names for waypoints used for the arrival instruction.
+   * @param waypointNames      custom names for waypoints used for the arrival instruction.
+   * @param bicycleType        The type of bicycle, either `Road`, `Hybrid`, `City`, `Cross`, `Mountain`.
```

Comment:
Are these values (`Road`, `Hybrid`, `City`, `Cross`, `Mountain`) case sensitive?

Reply:
What about declaring a `@StringDef` for ease of use?